core/src/test/java/dk/alexandra/fresco/framework/util/TestAesCtrDrbg.java

```diff
@@ -0,0 +1,135 @@
+package dk.alexandra.fresco.framework.util;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.fail;
+
+import java.util.Arrays;
+import java.util.Random;
+import org.junit.Test;
+
+public class TestAesCtrDrbg {
+
+  @Test
+  public void testAesCtrDrbg() {
+    Random rand = new Random(42);
+    byte[] seed = new byte[32];
+    rand.nextBytes(seed);
+    new AesCtrDrbg(seed);
+
+    try {
+      new AesCtrDrbg(new byte[33]);
+      fail("Constructor fail to throw exception on too long seed");
+    } catch (IllegalArgumentException e) {
```

Comment:
You can expect exceptions which would improve readability here

I.e. Test(expected=IllegalArgumentException.class)

Reply:
Sure, but then it would not be certain that all exceptions are thrown.